test/release/examples/PREDIFF

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-sort $2 > $2.prediff.tmp
+sed -e 's/\r$//' -e '/^salloc:/d' < $2 | sort > $2.prediff.tmp
```

Comment:
We shouldn't be tailoring this prediff to a specific launcher. If we need to filter out slurm commands we should use a global prediff